lib/saml/responses/base.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module SAML
+  module Responses
+    module Base
+      ERRORS = { clicked_deny:   { code: '001',
```

Comment:
errors clicked_deny, auth_too_late, auth_too_early and blank are only messages we see on login

Reply:
auth_too_late, auth_too_early, and blank can happen for logout too... clicked_deny can't, but thats fine, it will just never be a hash hit.